src/main/java/org/broadinstitute/hellbender/tools/exome/ReadCountCollection.java

```diff
@@ -72,30 +57,18 @@
      *     <li>{@code targets} contains any {@code null}},</li>
      *     <li>{@code columnNames} contains any {@code null}},</li>
      *     <li>{@code counts} contains any {@code null}},</li>
+     *     <li>{@code targets} contains any duplicates,</li>
+     *     <li>{@code columnNames} contains any duplicates,</li>
      *     <li>{@code targets} length does not match {@code counts} length or</li>
      *     <li>{@code counts} elements length does not match {@code columnNames} length</li>
      * </ul>
      */
-    public ReadCountCollection(final SetUniqueList<Target> targets, final SetUniqueList<String> columnNames, final RealMatrix counts) {
-        Utils.nonNull(targets,"the input targets cannot be null");
-        Utils.nonNull(columnNames,"the column names cannot be null");
-        Utils.nonNull(counts,"the counts cannot be null");
-        if (columnNames.contains(null)) {
-            throw new IllegalArgumentException("column names contain nulls");
-        } else if (targets.contains(null)) {
-            throw new IllegalArgumentException("there are some null targets");
-        } else if (counts.getRowDimension() != targets.size()) {
-            throw new IllegalArgumentException("number of count rows does not match the number of targets");
-        } else if (counts.getColumnDimension() != columnNames.size()) {
-            throw new IllegalArgumentException("number of count columns does not match the number of column names");
-        }
-        this.targets = Collections.unmodifiableList(new ArrayList<>(targets));
-        this.columnNames = Collections.unmodifiableList(new ArrayList<>(columnNames));
-        this.counts = counts.copy();
+    public ReadCountCollection(final List<Target> targets, final List<String> columnNames, final RealMatrix counts) {
+        this(targets, columnNames, counts, true);
     }
 
     /**
-     * Creates a new collection without verifying field values and without copying inputs.
+     * Creates a new collection or with or without verifying field values and without copying inputs.
```

Comment:
It copies if `verifyInput == true`, which is always the case when the entry point is the public constructor.